src/Simulation/QSharpFoundation/Math/Math.qs

```diff
@@ -97,8 +97,9 @@ namespace Microsoft.Quantum.Math {
     ///
     /// # Remarks
     /// See [System.Math.Log10](https://docs.microsoft.com/dotnet/api/system.math.log10) for more details.
-    function Log10 (input : Double) : Double {
-        body intrinsic;
+    function Log10(input : Double) : Double {
```

Comment:
LLVM has intrinsic for log base 10 (https://llvm.org/docs/LangRef.html#id2078). Calling it _might_ be faster and have better precision than converting the bases ourselves.

Reply:
I'm fine either way; @bettinaheim had highlighted this as something that could be made concrete instead of intrinsic, but I'm happy to undo that change if that would be preferred.

Reply:
There are also intrinsics for exp, sqrt and the like, but we first need to expose them via the .NET wrapper. I'd suggest to handle that here for now so that we can unblock e.g. the VQE example and start setting up more extensive testing, and then we can look into leveraging the llvm intrinsics directly.

Reply:
Sounds good, let me see if I can make some progress on `ModPowL`, `Round`, `PowD`, and `ExpD` as highlighted above, then. Thanks!

Reply:
Those should now be added, resolving. Thank you!